src/mmrelay/constants/config.py

```
@@ -59,6 +66,15 @@
 CONFIG_KEY_MSGS_TO_KEEP: Final[str] = "msgs_to_keep"
 CONFIG_KEY_WIPE_ON_RESTART: Final[str] = "wipe_on_restart"
 
+# Additional credential/config keys
+CONFIG_KEY_DEVICE_ID: Final[str] = "device_id"
+CONFIG_KEY_PASSWORD: Final[str] = (
+    "password"  # nosec B105 - config key name, not a password
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "CONFIG_KEY_PASSWORD"

(S105)